clickhouse/columns/date.cpp

```diff
@@ -113,4 +113,69 @@ ItemView ColumnDateTime::GetItem(size_t index) const {
     return data_->GetItem(index);
 }
 
+
+ColumnDateTime64::ColumnDateTime64()
+    : Column(Type::CreateDateTime64(3ul))
+    , data_(std::make_shared<ColumnDecimal>(18ul, 3ul))
+{}
+
+ColumnDateTime64::ColumnDateTime64(size_t precision)
+    : Column(Type::CreateDateTime64(precision))
+    , data_(std::make_shared<ColumnDecimal>(18ul, precision))
+{}
+
+
+void ColumnDateTime64::Append(const Int128& value) {
+    data_->Append(value);
+}
+
+void ColumnDateTime64::Append(const std::string& value) {
+    data_->Append(value);
+}
+
+
+Int128 ColumnDateTime64::At(size_t n) const {
```

Comment:
I Decided to use Int128 because ColumnDecimal deals with this type. Poco supports https://pocoproject.org/docs/Poco.DateTime.html, but DateTime64 in Clickhouse has a higher resolution than in Poco.